### PR TITLE
Add 0.191 release notes

### PR DESCRIPTION
**Pull request by dain**

### CODE REVIEW

File: presto-docs/src/main/sphinx/release/release-0.191.rst [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+* Fix regression that could cause high CPU usage for join queries when dictionary
+  processing for joins is enabled.
+* Fix :func:`bit_count` for bits between 33 and 63.
+* Add support for grouped join execution. When both sides of join
+  have the same table partitioning and the partitioning is addressable,
+  partial data can be loaded into memory at a time, making it possible to
+  execute the join with less peak memory usage. ``colocated_join`` must be
+  turned on, and ``concurrent_lifespans_per_task`` must be specified.
+* Allow connectors to report the amount of physical written data.
+* Add ability to dynamically scale the out the number of writer tasks rather
```
Comment (findepi) [inline presto-docs/src/main/sphinx/release/release-0.191.rst]: s/scale the out/scale out/

File: presto-docs/src/main/sphinx/release/release-0.191.rst [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+Release 0.191
+=============
+
+General Changes
+---------------
+
+* Fix regression that could cause high CPU usage for join queries when dictionary
+  processing for joins is enabled.
+* Fix :func:`bit_count` for bits between 33 and 63.
+* Add support for grouped join execution. When both sides of join
```
Comment (electrum) [inline presto-docs/src/main/sphinx/release/release-0.191.rst]: "of join" -> "of a join"

File: presto-docs/src/main/sphinx/release/release-0.191.rst [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+  have the same table partitioning and the partitioning is addressable,
+  partial data can be loaded into memory at a time, making it possible to
+  execute the join with less peak memory usage. ``colocated_join`` must be
```
Comment (electrum) [inline presto-docs/src/main/sphinx/release/release-0.191.rst]: This should mention both the config property and system property using the standard language (see writer scaling for an example)

File: presto-docs/src/main/sphinx/release/release-0.191.rst [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+CLI Changes
+-----------
+
+* Improve display of values of the Geometry type.
+
+Hive Changes
+------------
+
+* Add support for grouped join execution for Hive tables when both
+  side of a join have the same bucketing property.
```
Comment (electrum) [inline presto-docs/src/main/sphinx/release/release-0.191.rst]: side -> sides

**Pull request closed without merging**